### ISSUE TITLE
A question about metric choosing

### ISSUE THREAD

**Issue by coder-zjh** [automl/auto-sklearn#435]
Hello,I am a data engineer in an internet finance company. Now our team plans to use auto-sklearn to assist in our daily work, such as creating models to predict whether our clients will cheat or not. 
But before we decide to use auto-sklearn, we found that at least one problem trapped us and we hope if you could give us some suggestions.
For example,sometimes we evaluate a model according to its ks rather than its accuracy.If auto-sklearn takes accuracy as default metric to choose algorithms or tuning hyperparameters,the final result may not be expected.
So we want to know,in auto-sklearn, whether we could use another metric to evaluate the model or not? If we want to  get a model with the best KS performance in all the models which auto-sklearn trained,how can we do?

**Comment by coder-zjh** [comment]
Oh,i find the function 'make_scorer' in 'autosklearn/metrics/__init__.py',i guess that is the answer.

**Comment by mfeurer** [comment]
Great that you already found it yourself. You can also have a look at [this example](https://github.com/automl/auto-sklearn/blob/master/example/example_metrics.py).
